src/main/drivers/vtx_common.h

```diff
@@ -110,30 +110,12 @@ typedef struct vtxDeviceCapability_s {
     uint8_t bandCount;
     uint8_t channelCount;
     uint8_t powerCount;
+    uint8_t filler;
```

Comment:
What are the benefits of adding a filler for alignment even if there are no more members after it?

Reply:
It's just documentary, to alert anyone who makes later changes to the code.